docs/containers.conf.5.md

```diff
@@ -361,6 +361,11 @@ pod, we start a `pause` process in a container to hold open the namespaces
 associated with the  pod.  This container does nothing other then sleep,
 reserving the pods resources for the lifetime of the pod.
 
+**image_parallel_copies**=0
+
+Maximum number of image layers to be copied (pulled/pushed) simultaneously.
+Not setting this field, will fall back to containers/image defaults. (6)
```

Comment:
```suggestion
Not setting this field, or setting it to zero, will fall back to containers/image defaults.
```

Reply:
Just curious why you didn't take this one?